dask/dataframe/tests/test_shuffle.py

```diff
@@ -779,34 +793,6 @@ def test_compute_divisions():
         compute_divisions(c)
 
 
-# TODO: Fix sporadic failure on Python 3.8 and remove this xfail mark
```

Comment:
I've removed this test, but I can restore it if needed.

It's asserting that there are some files left around, but I'm not sure that we want that / can reliably assert that. My understanding was that we wanted them to be cleaned up automatically.